examples/is-prime/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+#![no_std]
+
+#[cfg(not(test))]
+#[panic_handler]
+fn my_panic(_info: &core::panic::PanicInfo) -> ! {
+    loop {}
+}
+
+/// Returns 1 if integer is prime
+#[no_mangle]
```

Comment:
```suggestion
```

`#[no_mangle]` is only needed for the `entrypoint` function because we are tracking it during the compilation.